src/middlewared/middlewared/alert/source/fips.py

```diff
@@ -0,0 +1,28 @@
+from datetime import timedelta
+
+from middlewared.alert.base import AlertClass, AlertCategory, Alert, AlertLevel, AlertSource, ProductType
+from middlewared.alert.schedule import IntervalSchedule
+
+
+class FIPSMisconfigurationAlertClass(AlertClass):
+    category = AlertCategory.SECURITY
+    level = AlertLevel.CRITICAL
+    title = "FIPS misconfiguration"
+    text = "FIPS is %(configuration)s, but FIPS provider is %(state)s."
+
+
+class FIPSProviderAlertSource(AlertSource):
+    schedule = IntervalSchedule(timedelta(hours=1))
+
+    products = (ProductType.ENTERPRISE,)
+    run_on_backup_node = False
+
+    async def check(self):
+        fips_configured = (await self.middleware.call('system.security.config'))['enable_fips']
+        fips_enabled = await self.middleware.call('system.security.info.fips_enabled')
```

Comment:
`fips_enabled` will raise a CallError in the event openssl binary exits with a returncode. We should probably catch the exception here so we don't show a traceback in UI?